apps/ewallet/lib/ewallet/policies/role_policy.ex

```diff
@@ -0,0 +1,23 @@
+defmodule EWallet.RolePolicy do
+  @moduledoc """
+  The authorization policy for roles.
+  """
+  @behaviour Bodyguard.Policy
+  alias EWalletDB.{Account, User}
+
+  # Any user can get a role
+  def authorize(:all, _user_or_key, _role_id), do: true
```

Comment:
Let's be more explicit here and use `admin_user:` or `key: `

Reply:
Updated. Please check.